src/main/java/com/github/discvrseq/walkers/VcfToLuceneIndexer.java

```diff
@@ -186,7 +186,7 @@ else if (line.getCountType() == VCFHeaderLineCount.R) {
                                 throw new GATKException("Incorrect number of annotations for " + infoField + ". Was: " + variant.getAttribute(infoField) + ", at " + variant.toStringWithoutGenotypes());
                             }
 
-                            Object val = vals.get(alleleIdx);
+                            Object val = vals.get(alleleIdx - 1);
```

Comment:
@hextraza are you sure about this? The list should contain REF and ALT alleles. These annotations are supposed to have one value per ALT, meaning the index is one less.